challenge_open/src/challenge_open.py

```diff
@@ -44,26 +40,6 @@ def execute(self, userdata):
         return "done"
 
 
-class StartDetectingSkeletonsForever(smach.State):
```

Comment:
If so, then in robot_smach_states, but for now here is fine. 

And actually it's kind of a weird thing, cuz' maybe this should be in a separate service or whatever to just get the latest upon request. 
For now, just keep that code in, I think.